### PR TITLE
[1LP][RFR][NOTEST] Remove insights and load balancers to fix test

### PR DESCRIPTION
**Pull request by valaparthvi**
## Purpose or Intent
Red Hat Insights is no longer present, remove all references
- __Fixing__ 
    1. cfme/tests/configure/test_landing_page.py
    2. test_delete_users_from_collection

### CODE REVIEW

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/configure/test_landing_page.py [hunk 1]
```diff
@@ -148,8 +147,6 @@
     "Optimize / Bottlenecks",
     "Optimize / Planning",
     "Optimize / Utilization",
-    "Red Hat Access Insights",
-    "Networks / Load Balancers",
```
Comment (john-dupuy) [inline cfme/tests/configure/test_landing_page.py]: Why are `Networks` being removed too? Is this to fix `cfme/tests/configure/test_landing_page.py`?

Comment (izapolsk) [reply cfme/tests/configure/test_landing_page.py]: good question

Comment (valaparthvi) [reply cfme/tests/configure/test_landing_page.py]: Network Load Balancers are no longer in 5.11, it has been deprecated.

**Pull request merged**